blueflood-elasticsearch/src/main/resources/events_mapping.json

```diff
@@ -0,0 +1,27 @@
+{
+  "graphite_event": {
+    "_routing": {
+      "required": true
+    },
+    "properties": {
+      "tenantId": {
+        "type": "string",
+        "index": "not_analyzed"
+      },
+      "what": {
+        "type": "string",
+        "index": "not_analyzed"
+      },
+      "data": {
+        "type": "string",
+        "index": "not_analyzed"
+      },
+      "tags": {
+        "type": "string"
```

Comment:
@chinmay-gupte yes we definitely need it for tags. I will change the file to reflect it.